ui/tests/acceptance/secrets/backend/totp/key-test.js

```diff
@@ -0,0 +1,173 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import authPage from 'vault/tests/pages/auth';
+import { click, fillIn, currentURL, visit, waitUntil } from '@ember/test-helpers';
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import { GENERAL } from 'vault/tests/helpers/general-selectors';
+import { SECRET_ENGINE_SELECTORS as SES } from 'vault/tests/helpers/secret-engine/secret-engine-selectors';
+import { mountBackend } from 'vault/tests/helpers/components/mount-backend-form-helpers';
+import { v4 as uuidv4 } from 'uuid';
+import { create } from 'ember-cli-page-object';
+import fm from 'vault/tests/pages/components/flash-message';
+
+const flashMessage = create(fm);
+
+const SELECTORS = {
+  menuItem: (item) => `[data-test-popup-menu="${item}"]`,
```

Comment:
since this could be a pattern reused more, it might be nice to add it [here](https://github.com/hashicorp/vault/blob/main/ui/tests/helpers/general-selectors.ts#L36) to the general-selectors